stargazer/stargazer.py

```diff
@@ -176,6 +178,14 @@ def reset_covariate_order(self):
         if self.original_cov_names is not None:
             self.cov_names = self.original_cov_names
 
+    def add_custom_lines(self, lines, location='footer top'):
+        for line in lines:
+            assert len(line) == self.num_models + 1, \
+                'Please input iterables of length "number of models + 1"'
```

Comment:
A separate `label` arg may be cleaner than requiring it to be `self.num_models + 1`. This would also be more future-proof, if `stargazer` eventually supports multiple left-most (non-model) columns.

Reply:
Maybe we can allow both approaches: check that ``line`` has ``self.num_models + 1`` (where 1 would be possibly replaced in the future by the number of left headers) items, and only if it has ``self.num_models`` instead, use the (optional) ``label`` argument (which would default to ``''``). (Raise if it has neither number of items)

Reply:
Great idea. I think only supporting `line` of length `self.num_models` with a `label` would be the cleanest option. That way we offer one clear interface. Offering both interfaces might be confusing, increases code complexity and still creates the multiple left-most columns problem that @MaxGhenis mentioned.

Reply:
> Great idea. I think only supporting `line` of length `self.num_models` with a `label` would be the cleanest option.

OK, no objection: then I would have ``label`` as first argument and  ``line`` as second.